canonical-websites/www.canonical.com
# Design feedback on /about

## Done

See design review: https://github.com/ubuntudesign/vanilla-squad/issues/58

## QA

- Pull code
- Run `./run serve --watch`
- Check http://0.0.0.0:8002/about against checklist above

## Issue / Card

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/58
